### DISCUSSION TITLE
i wanted to set up an ide i failed

### DISCUSSION THREAD

**Discussion by MohamedDhiaSleimi** [LazyVim/LazyVim#3677] [Q&A] [answered]
I am new to nvim (couple day) this is the fruit of 5hours of effort trying to set lazyvim as an ide i read the docs found other configs and copied from them but i don't know why it did not work 
could someone please tell me what i did wrong as far as i understand lua and what i read on the docs i don't have an obvious problem 
this is the ling to the config repo i pushed the entire nvim folder https://github.com/MohamedDhiaSleimi/lazyvim-config

**Comment by dotfrag** [comment, marked as answer]
Are you trying to setup python? There is an extra available for python already. When you open LazyVim, press `x` on the dashboard to load LazyExtras and enable the python extra.

**Reply by MohamedDhiaSleimi** [reply]
I don't have acess to my pc tight now i will look into it as soon as i could thank you

**Reply by MohamedDhiaSleimi** [reply]
yeah i found it .it did not work as good expected even in a language like python so i am gonna write code in vim and test it in vscode that is easier and even for quick fixes i can use vim mode for vscode.
thank you for the help i will mark it as the answer. maybe when I have the time to migrate to linux i will see if i can go vim full time i found more support for unix based systems than windows and thank you again

**Reply by dotfrag** [reply]
Is there something specific you didn't like? Maybe it can be customised, if it's not a limitation. I have used python for some projects without problems (i.e. same experience with VSCode), but I haven't tried big projects.

**Reply by MohamedDhiaSleimi** [reply]
It is nothing i am just too used to my current work flow to be able to change it in the middle of a project it is the same reason i did not migrate to linux yet  and thank you for the help when i start thinking about moving fully i will ask help here again if i need by then

**Reply by dotfrag** [reply]
No problem, good luck with your project and the potential of trying linux :)
